derive/src/decode.rs

```diff
@@ -20,11 +20,17 @@ use syn::{
 
 use crate::utils;
 
-pub fn quote(data: &Data, type_name: &Ident, input: &TokenStream) -> TokenStream {
+pub fn quote(
+	data: &Data,
+	type_name: &Ident,
+	// E.g. `<T, I>`
```

Comment:
Better to move this above as documentation

Reply:
yes doc improved